package.json

```diff
@@ -123,6 +123,6 @@
   },
   "dependencies": {
     "axe-webdriverjs": "2.3.0",
-    "lodash": "4.17.15"
```

Comment:
Our `lodash` dep is already above the required `4.17.12`, this change is a downgrade. I think the issue is with a peer dependency, this one? https://gyazo.com/11eb257275511290ef1393fdfdb7883c